template/python3-http-debian/function/handler.py

```diff
@@ -0,0 +1,5 @@
+def handle(event, context):
+    return {
+        "statusCode": 200,
+        "body": "Hello from OpenFaaS!"
```

Comment:
can we have this echo the request body, like in the default flask template handler

Reply:
Again, I'm matching the existing templates.  This is what the default http header does, so I don't want to change that. 

I don't want to change anything about these templates across the board. I just want to make the debian ones available and functional.

At least under this PR.